docs/functions/usage/index.md

```diff
@@ -54,7 +54,7 @@ Whilst developing your application with Cloud Functions, it is possible to run t
 To call the emulated functions, call the `useEmulator` method exposed by the library:
 
 ```js
-import functions from '@react-native-firebase/functions';
+import { firebase } from '@react-native-firebase/functions';
```

Comment:
This is not correct - it does not match the code below where `functions()` is usesd

Reply:
You are right. I reversed the change.